.editorconfig

```diff
@@ -0,0 +1,9 @@
+root = true
+
+[*]
+indent_style = space
+indent_size = 2
+end_of_line = lf
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
```

Comment:
It's not specific to atom it will make sure everyone's editor uses the same indenting and formatting style as we work on this project together.  

http://editorconfig.org/